CRM/Core/BAO/Phone.php

```diff
@@ -171,7 +171,9 @@ public static function allPhones($id, $updateBlankLocInfo = FALSE, $type = NULL,
   }
 
   /**
-   * Get all the phone numbers for a specified location_block id, with the primary phone being first
+   * Get all the phone numbers for a specified location_block id, with the primary phone being first.
+   *
+   * This is called from CRM_Core_BAO_Block as a calculated function.
```

Comment:
I don't think it is good form to say in comments where something is called from, as that can get dated.